environment.yml

```diff
@@ -0,0 +1,18 @@
+name: celer-env
+dependencies:
+  - python>=3.8
+  - pip
+  # dependencies
+  - Cython>=0.26
+  # - download
```

Comment:
those must be installed with pip. It's doable in this file, see https://stackoverflow.com/questions/35245401/combining-conda-environment-yml-with-pip-requirements-txt